packages/theme/index.js

```diff
@@ -0,0 +1,2 @@
+export { ThemeProvider } from './provider';
+export { defaultTheme } from './theme';
```

Comment:
These exports should probably be private, so that we can safely iterate over the APIs of the package while we're actively developing it


Also, why is the `defaultTheme` an export? Wouldn't consumers of the theme just read the CSS variables instead ?